src/LCT.Common/CycloneDXBomParser.cs

```diff
@@ -45,7 +47,7 @@ public Bom ParseCycloneDXBom(string filePath)
             {
                 Logger.Error("Exception in reading cycloneDx bom", ex);
             }
-            return bom;
+            return bom.Components;
```

Comment:
We can't take only BOM components details from here, because in future how will read other details of BOM(Ex: while reading SBOM template)

Reply:
implemented and reverted the class back